tensorflow/core/api_def/base_api/api_def_SamplingDataset.pbtxt

```diff
@@ -0,0 +1,12 @@
+op {
+  graph_op_name: "SamplingDataset"
+  in_arg {
+    name: "rate"
+    description: <<END
+A scalar representing the sample rate of elements from the `input_dataset`
+that should be taken.
+is taken.
```

Comment:
this looks like a typo

Reply:
Fixed in https://github.com/tensorflow/tensorflow/pull/25384/commits/fa11069c3e85ffbcae19994bdba09c30eee0a6de#diff-10942c7d6fdf513b324aefbdcc44cedaL8